app/models/public_body.rb

```diff
@@ -64,6 +64,8 @@ class ImportCSVDryRun < StandardError ; end
   validates_presence_of :url_name, :message => N_("URL name can't be blank")
   validates_presence_of :last_edit_editor,
                         :message => N_("Last edit editor can't be blank")
+  validates_presence_of :last_edit_comment,
```

Comment:
This should be a `not_nil` validation, as the database has records where the comment is empty string.